### PR TITLE
Fix: no rows in query panel

### PR DESCRIPTION
**Pull request by AdiGajbhiye**
## Checklist

- [ ] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

File: query_panel/main.js [hunk 1]
```diff
@@ -16,16 +16,26 @@ async function updateConfig(config) {
 const DEFAULT_HEIGHT = 455;
 
 class Grid {
+  table;
```
Comment (mdesmet) [inline query_panel/main.js:19]: `constructor(private table)`?

File: query_panel/main.js [hunk 2]
```diff
@@ -43,8 +53,7 @@ window.addEventListener("DOMContentLoaded", async function () {
 const app = createApp({
   data() {
     return {
-      count: 0,
-      data: null,
+      eventData: null,
```
Comment (mdesmet) [inline query_panel/main.js:56]: We have been changing the name of this field lots of times. I'm not sure if the new name is better, why not keep the name data?

File: query_panel/main.js (hunk 1 above)
Comment (mdesmet) [reply query_panel/main.js:19]: Why do we need to even save the table reference?

File: query_panel/main.js [hunk 3]
```diff
@@ -50,9 +50,10 @@ class Grid {
       schema[result.columnNames[i]] = this.mapType(result.columnTypes[i]);
     }
     const table = await this.worker.table(schema);
-    table.replace(result.rows);
+    await table.replace(JSON.parse(JSON.stringify(result.rows)));
```
Comment (lionsardesai) [inline query_panel/main.js:53]: why was this needed? should we be catching anything if this fails?

**Comment by AdiGajbhiye** [comment]
closing in favor of https://github.com/AltimateAI/vscode-dbt-power-user/pull/657

**Pull request closed without merging**